src/test/java/org/candlepin/resource/ConsumerResourceCreationTest.java

```diff
@@ -497,7 +532,8 @@ public void cannotMixUsernameWithActivationKeys() {
         when(this.principalProvider.get()).thenReturn(p);
 
         assertThrows(BadRequestException.class,
-            () -> resource.createConsumer(consumer, USER, owner.getKey(), createKeysString(keys), true));
+            () -> resource.createConsumer(consumer, USER, owner.getKey(), createKeysString(keys),
+            true, null));
```

Comment:
Carry-on line should be indented

Reply:
i know but there was problem with checkstyle so i reformat it different way. done